src/main/resources/db/migration/V1_18__create_jrd_lrd_region_mappings.sql

```diff
@@ -0,0 +1,102 @@
+---- new table created : jrd_lrd_region_mapping
+CREATE TABLE jrd_lrd_region_mapping(
+    jrd_region_id VARCHAR(16) NOT NULL,
+    jrd_region VARCHAR(256) NOT NULL,
+    region_id VARCHAR(16) NOT NULL,
+    region VARCHAR(256) NOT NULL
+);
+
+---- inserts for table : jrd_lrd_region_mapping
```

Comment:
you can have one single insert statement with multiple values
`INSERT INTO <TableName> ( Column1, Column2 ) VALUES
( Value1, Value2 ), ( Value1, Value2 )`

Reply:
@SujeethKumarAri : If you generate your inserts through excel, you do not have the provision of keeping the insert statement into single line.

Reply:
@kavithagade12  - yes but we need to edit it to have single insert statement with multiple values which will improve the performance
https://stackoverflow.com/questions/1793169/which-is-faster-multiple-single-inserts-or-one-multiple-row-insert#:~:text=If%20you%20are%20inserting%20many,separate%20single%2Drow%20INSERT%20statements.

Reply:
Its been updated. Please review. Thank you.